README.md

```diff
@@ -128,8 +120,8 @@ Si encuentra un error, informe los errores grandes y pequeños [**abriendo un is
 
 <p align="center">
 	<a href="https://eoscostarica.io/">
-		<img src="https://raw.githubusercontent.com/eoscostarica/design-assets/master/logos/eosCR/fullColor-horizontal-transparent-white.png"
-		width="400px" >
+		<img src="https://user-images.githubusercontent.com/5632966/178800859-e05a07ff-b3e5-41a9-9137-d1a271b958ff.png"
```

Comment:
lets source this image from the same location as the other logos `.github` folder

Reply:
fixed.